packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -379,8 +390,14 @@ abstract class ImageProvider<T> {
 
   /// Converts a key into an [ImageStreamCompleter], and begins fetching the
   /// image.
+  ///
+  /// The [decode] callback provides the logic to obtain the codec for the
+  /// image as well as a way to supply parameters as a closure.
```

Comment:
i don't know if anyone will understand "as well as a way to supply parameters as a closure".